index.html

```diff
@@ -241,15 +299,30 @@
         Security and Privacy Considerations
       </h2>
       <p>
+        Due to differences in "quality of implementation", and the end-user's
```

Comment:
Due to differences in "quality of implementation", and the end-user's

I think it’s not necessary to put _quality of implementation_ in quotes here